docs/source/tracking.rst

```diff
@@ -1015,6 +1018,10 @@ Also, you must run ``pip install azure-storage-blob`` separately (on both your c
 Finally, if you want to use DefaultAzureCredential, you must ``pip install azure-identity``;
 MLflow does not declare a dependency on these packages by default.
 
+You may set an MLflow environment variable to configure the timeout for artifact uploads and downloads:
+
+- ``MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT`` - (Experimental, may be changed or removed) Sets the timeout for artifact upload/download in seconds (Default: 600).
```

Comment:
Since each store can decide on a default, can we update the docstring to indicate that, if this isn't set, the upload timeout depends on the artifact storage backend?

Reply:
The generic description for this env var says that. I've updated it in the Azure (here) and GCS sections to make it clear that those defaults are store-based.